src/CatalogPage.js

```diff
@@ -1,10 +1,53 @@
+import { it } from "node:test";
+
 /**
  * This class obtains the elements in the DOM for the
  * catalog page using different locators
  */
 class CatalogPage {
   constructor(page) {
     this.productTitle = page.getByTestId("title");
+    this.productPrices = page.getByTestId("inventory-item-price");
+    this.dropdown = page.locator("//select[@data-test='product-sort-container']");
   }
+
+  async getPriceListArray(){
+    const priceItemsArray = [];
+    for(let item of await this.productPrices.all()){
+      const price  = await item.innerText();
+      const priceFloat = parseFloat(price.replace("$", ""));
+      priceItemsArray.push(priceFloat);
+    }
+    return priceItemsArray;
+  }
+
+  async orderArrayAscending(itemsPriceArray){
+    var pricceItemOrdered = itemsPriceArray.sort(function (a,b) {
+      return a - b; // Ascending
+  });;
+  return pricceItemOrdered;
+  }
+
+  async selectOrderBy(){
+    await this.dropdown.selectOption('Price (low to high)');
+  }
+
+  async compareArrays(pricesOrdered, pricesOrderByWebPage){
```

Comment:
This method is a good candidate to be in a` utils.js` file by itself instead of being part of a Page Object Model.

It is a method you could use anywhere in your Automation solution whenever you need to compare two arrays.